__tests__/app.test.js

```diff
@@ -120,3 +120,44 @@ describe("/api/articles/nickiminaj", () => {
       });
   });
 });
+describe("200 - GET: /api/articles/:article_id/comments", () => {
+  it("should return an array of comments with that comment ID ordered by created date descending", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const commentsArr = response.body.comments;
+
+        expect(commentsArr).toBeSorted({
+          key: "created_at",
+          descending: "true",
+        });
+      });
+  });
+  it("Should respond with an array of objects matching expected properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const commentsArr = response.body.comments;
+        commentsArr.forEach((comment) => {
+          expect(comment).toEqual({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 1,
+          });
+        });
+      });
+  });
+  it("Should respond with not found obj if queried with a valid yet non existent id", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "Comment not found" });
+      });
+  });
+});
```

Comment:
Could we just add the following two tests here please?
- Status 400, invalid ID, e.g. string of "not-an-id"
-Status 200, valid ID, but has no comments responds with an empty array of comments